symfony/plugins/orangehrmRESTPlugin/lib/Api/Attendance/PunchTimeAPI.php

```diff
@@ -106,4 +109,20 @@ function getTimezoneOffset($remote_tz, $origin_tz = null) {
         return $offset;
     }
 
-}
\ No newline at end of file
+    public function getPunchTimeEditable()
+    {
+        $isPunchTimeEditable = $this->getAttendanceService()->getSavedConfiguration(
+            WorkflowStateMachine::FLOW_ATTENDANCE,
+            AttendanceRecord::STATE_INITIAL,
+            configureAction::ESS_USER,
+            WorkflowStateMachine::ATTENDANCE_ACTION_EDIT_PUNCH_TIME,
+            AttendanceRecord::STATE_INITIAL
+        );
+        $serverUtcTime = $serverUtcTime = gmdate('Y-m-d H:i');
```

Comment:
`$serverUtcTime  = gmdate('Y-m-d H:i');`  ??

Reply:
yh.. I will correct it